students/navdeep/session06/mailroom_part4.py

```diff
@@ -0,0 +1,149 @@
+import os.path
+
+def menu(donor_list):
+    """This function will print out a menu of options for
+    the user to choose from
+    :param: the list of donors and how much they have donated"""
+    user_option = ""
```

Comment:
You could probably simplify your function by forcing user_option to .lower() right away here.